Origin Webpage,Status Code / Error,URL
https://auto.gluon.ai/dev/_sources/tutorials/multimodal/advanced_topics/hyperparameter_optimization.ipynb.txt ,403,"https://auto.gluon.ai/dev/_sources/tutorials/multimodal/advanced_topics/\\""https:/docs.ray.io/en/latest/train/api/doc/ray.train.CheckpointConfig.html#ray.train.CheckpointConfig\\"""
https://auto.gluon.ai/dev/tutorials/multimodal/document_prediction/document_classification.html ,404,https://huggingface.co/datasets/rvl_cdip
https://auto.gluon.ai/dev/tutorials/tabular/tabular-indepth.html#specifying-hyperparameters-and-tuning-them ,403,https://dl.acm.org/doi/pdf/10.1145/3236386.3241340
https://auto.gluon.ai/dev/whats_new/v0.4.0.html ,404,https://github.com/truebluejason
https://auto.gluon.ai/dev/tutorials/timeseries/advanced/index.html ,403,https://auto.gluon.ai/dev/tutorials/timeseries/advanced/advanced/forecasting-custom-model.html
https://auto.gluon.ai/dev/whats_new/v0.8.0.html ,404,https://github.com/zhiqaingdon
https://auto.gluon.ai/dev/tutorials/multimodal/object_detection/data_preparation/prepare_voc.html ,<urlopen error timed out>,http://host.robots.ox.ac.uk/pascal/VOC/
https://auto.gluon.ai/dev/api/autogluon.tabular.TabularPredictor.simulation_artifact.html ,404,https://github.com/autogluon/tabrepo/blob/main/examples/run_quickstart_from_scratch.py
https://auto.gluon.ai/dev/tutorials/tabular/tabular-essentials.html ,<urlopen error [Errno 101] Network is unreachable>,https://www.gnu.org/software/wget/
https://auto.gluon.ai/dev/whats_new/v0.5.1.html ,404,https://huggingface.co/datasets/paws-x
https://auto.gluon.ai/dev/whats_new/v0.5.1.html ,404,https://huggingface.co/datasets/stsb_multi_mt
https://auto.gluon.ai/dev/whats_new/v0.5.1.html ,404,https://huggingface.co/datasets/amazon_reviews_multi
